tests/CacheCompat/CommonCache.Test.MsalNode/NodeScript/index.js

```diff
@@ -0,0 +1,180 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+
+/* To run this locally, start with `node index.js --inputPath c:\\temp\\input.json` where input.json contains
+
+{
+    "Scope": "https://graph.microsoft.com/user.read",
+    "CacheFilePath": "C:\\Users\\bogavril\\AppData\\Local\\Temp\\adalcachecompattestdata\\msalCacheV3.binx",
+    "LabUserDatas": [
+        {
+            "Upn": "liu.kang@bogavrilltd.onmicrosoft.com",
+            "Password": "****",
+            "ClientId": "1d18b3b0-251b-4714-a02a-9956cec86c2d",
+            "TenantId": "49f548d0-12b7-4169-a390-bb5304d24462",
+            "Authority": "https://login.microsoftonline.com/49f548d0-12b7-4169-a390-bb5304d24462/"
+        }
+    ],
+    "ResultsFilePath": "C:\\Users\\bogavril\\AppData\\Local\\Temp\\adalcachecompattestdata\\msal_python_results.json",
```

Comment:
? - several of these around this code base

Reply:
Not sure what the problem is. It's an example. Folks can uncomment and use their own path to run the script.